src/System Application/App/Language/src/Language.Codeunit.al

```diff
@@ -217,6 +217,48 @@ codeunit 43 Language
         LanguageImpl.SetPreferredLanguageID(UserSecID, NewLanguageID);
     end;
 
+    /// <summary>
+    /// Retrieves the two-letter ISO language name for the specified language ID.
+    /// </summary>
+    /// <param name="LanguageID">The language ID.</param>
+    /// <returns>The two-letter ISO language name.</returns>
+    procedure GetTwoLetterISOLanguageName(LanguageID: Integer): Text
+    var
+        LanguageImpl: Codeunit "Language Impl.";
+    begin
+        exit(LanguageImpl.GetTwoLetterISOLanguageName(LanguageID));
+    end;
+
+    /// <summary>
+    /// Retrieves the language ID for the specified culture name.
+    /// </summary>
+    /// <param name="CultureName">The Culture name.</param>
+    /// <returns>The language ID.</returns>
+    /// <example>
+    /// <code>
+    /// Language.GetLanguageIdFromLanguageName('en-US');
+    /// Language.GetLanguageIdFromLanguageName('en');
+    /// </code>
+    /// </example>
+    procedure GetLanguageIdFromCultureName(CultureName: Text): Integer
```

Comment:
Can we point to some docs indicating how to understand these ids? I get an integer but what does the different values indicate?

Reply:
Next PR